execution_workflows/LABRAT/modules/labrat_maketffasta.nf

```diff
@@ -8,11 +8,14 @@ process LABRAT_MAKETFFASTA {
         publishDir "${params.outdir}/labrat", mode: params.publish_dir_mode
         container "quay.io/biocontainers/labrat:0.2.2--pyhdfd78af_0"
 
+        //time '8h'
```

Comment:
does this comment mean the time taken to run this particular process is 8h or what does this comment mean?

Reply:
yes, this process takes more than the default 4h, so it would fail. @yuukiiwa commented it out. 
I can remove it altogether.